Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -864,9 +775,6 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 		u32 pteg_addr = ((hash & PowerPC::ppcState.pagetable_hashmask) << 6) | PowerPC::ppcState.pagetable_base;
 
-		if ((pteg_addr >> 28) == 1)
-			base_mem = Memory::m_pEXRAM;
-
```

Comment:
This bit of code arrived from a note by @tueidj pointing out that pteg's in mem2 wouldn't work.  pteg_addr >> 28 == 1 is a way of checking if the pteg_addr is in 0x1XXXXXXX (the physical address of mem2).  This code should be kept and any base_mem accesses should be masked with the RAM_MASK or EXRAM_MASK as appropriate.